scripts/update_apkid.sh

```diff
@@ -0,0 +1,30 @@
+script_path=$(dirname $0)
+if [ "$script_path" != "scripts" ] && [ "$script_path" != "./scripts" ]; then
+    echo 'Please run script from MobSF directory '
+    echo './scripts/update_apkid.sh '
+    exit  1
+fi
+if [ ! -d venv/bin/ ]; then
+   virtualenv venv -p python3
+   source venv/bin/activate
```

Comment:
Is this source and venv check required? We already sourced from the parent script.
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/88646375b1180dd5ff9aed4fb6dd417e91ce4ac8/setup.sh#L11